ScoutSuite/providers/azure/services/securitycenter.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+from ScoutSuite.providers.azure.configs.base import AzureBaseConfig
+
+
+class SecurityCenterConfig(AzureBaseConfig):
+    targets = (
+        ('pricings', 'Pricings', 'list', {}, False),
+        ('security_contacts', 'Security Contacts', 'list', {}, False),
+        ('auto_provisioning_settings', 'Auto Provisioning Settings', 'list', {}, False)
+    )
+
+    def __init__(self, thread_config):
+        self.pricings = {}
+        self.pricings_count = 0
+
+        self.security_contacts = {}
+        self.security_contacts_count = 0
+
+        self.auto_provisioning_settings = {}
+        self.auto_provisioning_settings_count = 0
+
+        super(SecurityCenterConfig, self).__init__(thread_config)
+
+    def parse_pricings(self, pricing, params):
+        pricing_dict = {'id': pricing.id,
+                        'name': pricing.name,
+                        'pricing_tier': pricing.pricing_tier}
+
+        self.pricings[pricing_dict['id']] = pricing_dict
+
+    def parse_security_contacts(self, security_contact, params):
+        security_contact_dict = {
```

Comment:
The current codebase defines resource dictionaries as following:

```python
security_contact_dict = {}
security_contact_dict['id'] = security_contact.id
security_contact_dict['name'] = security_contact.name
security_contact_dict['email'] = security_contact.email
# ...
```
Maybe we should keep it this way for code consistency? (Also this way may be more resilient in case of API changes, allowing the resource dictionary to be created and partially populated anyway, even if some of the attributes are not retrievable anymore.)

Reply:
I just had the warning that this was not PEP8 compliant, and I honestly don't think it would be more resiliant(just comment out a line in both case to remove something). Altough, code consistency is a good argument

Reply:
I would stay consistent with the current codebase for now and consider refactoring it everywhere later on!